### ISSUE TITLE
Windows support.

### ISSUE THREAD

**Issue by NoPantsCrash** [NoPantsCrash/ffenmass#3]
Make FFenmass cross compatible with Windows pathing systems

**Comment by NoPantsCrash** [comment]
Things that need to happen first to make this work.
- Get a working windows machine.
- Make FFenmass somehow handle extensions, or else windows will have issues with media files without extensions.
- Redo the Directory mirroring to work with windows paths.

**Comment by NoPantsCrash** [comment]
Postponed until further notice.
